.github/workflows/cronJob.yaml

```diff
@@ -0,0 +1,53 @@
+name: Cron Job
+
+on:
+  workflow_dispatch:
+  schedule:
+    # The job runs every midnight (UTC time). Ref - https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onschedule
+    - cron: '0 0 * * *'
+
+jobs:
+  fetch_all_pull_request_shas:
+    runs-on: ubuntu-latest
+    outputs:
+      shas: ${{ steps.extract.outputs.shas }}
+    steps:
+      - name: Extract merge_commit_shas
+        shell: bash
+        id: extract
+        run: |
+          pr_infos=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+          "https://api.github.com/repos/redhat-developer/lsp4ij/pulls?&state=open&sort=created&direction=desc&per_page=100")
```

Comment:
Please add this api url as a parameter instead of hardcoding it here.

Reply:
sort=created is not needed since it is sorted as created by default, so removed sort. Also state is open, direction is desc and per_page=30 by default. Removed that as well.

Reply:
Added the api url as parameter